### ISSUE TITLE
When installing charts in a Rancher deployed environment with proxied nodes, the terminal results in `There are no log entries in the current range`

### ISSUE THREAD

**Issue by markusewalker** [rancher/rancher#37698]
**Rancher Server Setup**
- Rancher version: `v2.6.5-rc11`
- Installation option (Docker install/Helm Chart): Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE2 `v1.23.6`
- Proxy/Cert Details:

**Information about the Cluster**
- Kubernetes version: `v1.23.6`
- Cluster Type (Local/Downstream): Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): Custom

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom) Standard
  - If custom, define the set of permissions:

   
**Describe the bug**

I am using the following setup: Rancher deployed without a proxy, custom cluster with nodes running behind a proxy. When attempting to install any type of chart, the terminal opens, but nothing shows up. What is expected is a list of commands and their results indicating success or failure. Instead, I get this in the terminal:

![image](https://user-images.githubusercontent.com/52591819/167959062-60610b4f-5b09-4243-9098-06d052bf97a5.png)

This is not expected output. Looking in the browser developer tools, I am seeing the following:

![Screen Shot 2022-05-11 at 3 40 51 PM](https://user-images.githubusercontent.com/52591819/167959205-1d39768a-d171-4229-9ceb-953fdeb22ab5.png)

**To Reproduce**
 
1. Setup Rancher and navigate to Rancher UI in a browser.
2. Create standard user and login to Rancher as that user.
3. Create a custom cluster with nodes in a proxied environment.
4. Navigate to Cluster Explorer > cluster > Apps > Charts.
5. Install Monitoring (or any other chart should be fine); verify that nothing appears in the terminal.

**Result**
No output is given in the terminal as you begin to install the chart, however the chart will deploy successfully.

**Expected Result**

The resulting terminal should output each command and step prior to the chart being deployed.

**Comment by cbron** [comment]
>  Rancher deployed without a proxy, custom cluster with nodes running behind a proxy. 

So Rancher and its nodes are not behind a proxy, and a downstream custom cluster has all its nodes behind a proxy, is that right ? 
Can you also clarify what kind of proxy.

**Comment by markusewalker** [comment]
@cbron that is correct; Rancher and nodes are not behind a proxy, but the downstream custom cluster and nodes are behind a proxy. This is an HTTP proxy configured within AWS.

**Comment by MKlimuszka** [comment]
I think this was incorrectly labeled as team 3. Labeling it as team 1

**Comment by samjustus** [comment]
We think this is team 3 (or UI), @MKlimuszka 
it seems to be related to the helm installer logs.

**Comment by markusewalker** [comment]
Despite this issue being unscheduled, here is the current QA test plan for team 3 whenever this issue is gotten around to:

**TEST PLAN**
1. Setup Rancher and navigate to Rancher UI in a browser.
2. Create standard user and login to Rancher as that user.
3. Create a custom cluster with nodes in a proxied environment.
     - Rancher itself is NOT behind a proxy, only the custom cluster and it's respective nodes ARE behind a proxy.
5. Navigate to Cluster Explorer > cluster > Apps > Charts.
6. Install Monitoring (or any other chart should be fine); verify that nothing appears in the terminal.
